### PR TITLE
Add `--non-interactive` flag

### PR DESCRIPTION
**Pull request by JakubMatejka**

### CODE REVIEW

File: cmd/kbc/main.go [hunk 1]
```diff
@@ -5,13 +5,13 @@ import (
...
 	"github.com/keboola/keboola-as-code/internal/pkg/filesystem/clifs"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/cli"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/cli/cmd"
-	interactivePrompt "github.com/keboola/keboola-as-code/internal/pkg/service/cli/prompt/interactive"
 )
 
 func main() {
 	// Interactive prompt
-	prompt := interactivePrompt.New(os.Stdin, os.Stdout, os.Stderr)
+	p := cli.NewPrompt(os.Stdin, os.Stdout, os.Stderr)
```
[the remark this answers is not in the document]
Comment (michaljurecko) [reply cmd/kbc/main.go:14]: Prvy krat sa pouzije tu:
https://github.com/keboola/keboola-as-code/blob/1d4961ace5e7dfd85d7acd5e7e784210cf209f60/internal/pkg/service/cli/cmd/cmd.go#L154-L155

Teda mohli by sme ho vytvorit nad riadkom `154`, tam by mali byt dostupne uz aj flags.

------------

Nasie som tem prompt iba v jednom teste:
https://github.com/keboola/keboola-as-code/blob/1d4961ace5e7dfd85d7acd5e7e784210cf209f60/internal/pkg/service/cli/cmd/cmd_test.go#L227

A tam sa ziadne dialogy netestuju, takze by malo stacit vyhodit ten parameter.

Ak by to neslo napis, najdem riesenie (napr. sa nastavi `envs.Set("KBC_NON_INTERACTIVE", "true")`.

File: internal/pkg/service/cli/dialog/branches_test.go [hunk 2]
```diff
@@ -75,6 +75,25 @@ func TestSelectBranchByFlag(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func TestSelectBranchNonInteractive(t *testing.T) {
```
Comment (JakubMatejka) [inline internal/pkg/service/cli/dialog/branches_test.go:78]: Tak to funguje. Přidal jsem na to tyhle dva testy, to by mohlo stačit?

File: cmd/kbc/main.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (JakubMatejka) [reply cmd/kbc/main.go]: Jo pravda, jde to.

File: test/cli/help/workspace-list/expected-stdout [hunk 3]
```diff
@@ -9,7 +9,8 @@ Flags:
 Global Flags:
   -h, --help                       print help for command
   -l, --log-file string            path to a log file for details
+      --non-interactive            disable user interaction
```
Comment (michaljurecko) [inline test/cli/help/workspace-list/expected-stdout:12]: Toto by bolo mozno dobre napisat nejak jasnejsie, co napriklad: `disable user interaction` -> `disable interactive dialogs`?

**Review by michaljurecko — APPROVED** [review]
LGTM

File: test/cli/help/workspace-list/expected-stdout (hunk 3 above)
Comment (JakubMatejka) [reply test/cli/help/workspace-list/expected-stdout:12]: jo, pravda. upravím

**Pull request merged**